orc8r/cloud/deploy/bare-metal-ansible/README.md

```diff
@@ -0,0 +1,46 @@
+Deploying Magma on bare metal
+
+The following file needs to be edited before deployment can start:
+* ansible_vars.yaml
+
+The values that need to be customized are the IP settings for your network and
+the passwords which need to be generated. Additionally, you need to host a
+docker repo and helm chart repo for orc8r. Further details can be found at
+https://magma.github.io/magma/docs/orc8r/deploy_build#build-and-publish-helm-charts
+
+The deployment assumes the host running Ansible and the target hosts are
+running Ubuntu 18.04 or 20.04.
+
+The following variables must be set before starting:
+* orc8r_image_repo - Docker repo where controller, magmalte, and nginx containers are located
+* orc8r_helm_repo - Helm chart repo where orc8r chart is located
+* orc8r_domain - DNS domain where Magma services are published
+* orc8r_nms_admin_email - Initial login user for NMS
+* orc8r_chart_version - Helm chart version for orc8r
+* orc8r_image_tag - orc8r controller Docker image tag
+* orc8r_nms_image_tag - NMS magmalte Docker image tag
+* orc8r_nginx_image_tag - orc8r nginx Docker image tag
+* metallb_addresses - An IP range of at least 5 IP addresses for LoadBalancers for Magma
+
+If your environment requires it, the following variables should be set:
+* ansible_user - ssh user for your k8s hosts
+* loadbalancer_apiserver - contains port and address where to set a loadbalancer
+* vrrp_nic - network interface which keepalived should bind to on all hosts
+* docker_insecure_registries - If your orc8r_image_repo is insecure, it should be configured here
+* db_root_password - MariaDB root password (randomly generated if not specified)
```

Comment:
Though not enforced strictly, the preferred default database option is Postgres. Although NMS uses interfaces that would work for both mariadb and postgres, there are capabilities available in postgres (such as skip locked) which do not exist in mariadb. If you are already looking at single DB deployments for the on-prem installer, postgres might be a better option. cc @hcgatewood

Reply:
@arunuke  postgres was not working for a very long time https://github.com/magma/magma/issues/2896
The fix is only present in master and will not likely be backported to 1.3 (and possibly not even to 1.4). We might have to wait until Magma 1.5 release to use Postgres. I am very much in favor of deploying Postgres when it becomes realistic.